solutions/cf-1557B.mdx

```diff
@@ -259,7 +259,8 @@ void solve() {
 	next[sarr[n-1]] = MOD; // Ridiculously Large Number
 	int ans = 1;
 	for (int i = 1; i < n; i++) {
-		// If the previous element in the unsorted array is not the same as the previous element in the sorted array, we need at least 1 more segment
+		// If the previous element in the unsorted array is not the same 
+		// as the previous element in the sorted array, we need at least 1 more segment
```

Comment:
```cpp
/*
 * remember to style multiline
 * comments something like this
 */
```

Reply:
For 2 lines it is simpler tonot use /* * */

Reply:
actually yeah nvm